docs/concepts/workloads/vm.md

```diff
@@ -467,3 +467,118 @@ rockylinux_64Guest          Rocky Linux (64-bit)
 windows2022srvNext_64Guest  Microsoft Windows Server 2025 (64-bit)
 ...
 ```
+
+## Crypto
+
+The field `spec.crypto` may be used in conjunction with a VM's storage class and/or virtual trusted platform module (vTPM) to control a VM's encryption level.
+
+### Encrypting a VM
+
+#### Encryption type
+
+The type of encryption depends on the storage class and hardware present in the VM as the chart below illustrates:
+
+|                          | Config | Disks |
+|--------------------------|:------:|:-----:|
+| Encryption storage class |    ✓   |   ✓   |
+| vTPM                     |    ✓   |       |
+
+#### Default Key Provider
+
+By default, `spec.crypto.useDefaultKeyProvider` is true, meaning that if there is a default key provider and the VM has an encryption storage class and/or vTPM, the VM will be subject to some level of encryption. For example, if there was a default key provider present, it would be used to encrypt the following VM:
+
+```yaml
+apiVersion: vmoperator.vmware.com/v1alpha3
+kind: VirtualMachine
+metadata:
+  name: my-vm
+  namespace: my-namespace
+spec:
+  className:    my-vm-class
+  imageName:    vmi-0a0044d7c690bcbea
+  storageClass: my-encrypted-storage-class
+```
+
+#### Encryption Class
+
+It is also possible to set `spec.crypto.encryptionClassName` to the name of an `EncryptionClass` resource in the same namespace as the VM. This resource specifies the provider and key ID used to encrypt workloads. For example, the following VM would be deployed and encrypted using the `EncryptionClass` named `my-encryption-class`:
+
+```yaml
+apiVersion: vmoperator.vmware.com/v1alpha3
+kind: VirtualMachine
+metadata:
+  name: my-vm
+  namespace: my-namespace
+spec:
+  className:    my-vm-class
+  imageName:    vmi-0a0044d7c690bcbea
+  storageClass: my-encrypted-storage-class
+  crypto:
+    encryptionClassName: my-encryption-class
+```
+
+### Rekeying a VM
+
+Users can rekey VMs by switching to a different `EncryptionClass` or setting a currently non-empty `spec.crypto.encryptionClassName` to an empty value. This will result in the VM being rekeyed using the default key provider if it exists. If it does not, then the VM could be left in a state where it cannot be powered on.
+
+### Decrypting a VM
+
+It is not possible to decrypt an encrypted VM using VM Operator.
+
+### Status
+
+#### Observed type, provider, & key
+
+The following fields may be used to determine the encryption status of a VM:
+
+| Name | Description |
+|------|-------------|
+| `status.crypto.encrypted` | The observed state of the VM's encryption. May be `config`, `disks`, or both. |
+| `status.crypto.providerID` | The provider ID used to encrypt the VM. |
+| `status.crypto.keyID` | The key ID used to encrypt the VM. |
+
+For example, the following is an example of the status of a VM encrypted with an encrypted storage class:
```

Comment:
I noticed you have a mixed use of "encryption storage class" and "encrypted storage class" here and also in the code. Should we stick to one term ("encryption storage class") to keep it consistent with the API field?